rapid7_insightidr/plugin.spec.yaml

```diff
@@ -494,3 +494,53 @@ actions:
         description: Events from logs
         type: "[]events"
         required: true
+  close_investigations_in_bulk:
+    title: Close Investigations in Bulk
+    description: Close all investigations that match the parameters
+    input:
+      source:
+        title: Source
+        description: The name of an investigation source
+        type: string
+        example: MANUAL
+        default: MANUAL
+        required: false
+        enum:
+          - ALERT
+          - MANUAL
+          - HUNT
+      datetime_from:
+        title: Datetime From
+        description: An ISO formatted timestamp, default last week
+        type: string
+        example: 2018-07-01T00:00:00Z
+        required: false
+      datetime_to:
+        title: Datetime To
+        description: The category of alerts that should be closed, when empty use now
```

Comment:
This doesn't seem right - should be `An ISO formatted timestamp of the ending date range`